Sources/Controllers/Map/OAMapHudViewController.mm

```diff
@@ -1024,6 +1050,21 @@ - (void) updateButtonsLayout:(BOOL)animated
     }
 }
 
+- (void) layoutButtonsToStreet: (UIView*) container animated: (BOOL) animated
+{
+    CGFloat xLand = container.frame.origin.x;
+    CGFloat yLand = container.frame.origin.y + container.frame.size.height;
+    if (animated){
```

Comment:
We use standard C formatting, the curly braces should be written in the next line